.github/workflows/call-calc-coverage.yml

```diff
@@ -22,3 +22,6 @@ on:
 jobs:
   call-workflow:
     uses: nmfs-fish-tools/ghactions4r/.github/workflows/calc-coverage.yml@main
+    secrets:
```

Comment:
Do we want to add brief documentation or a link to instructions on how to add an upload token as a secret? Additionally, maybe include a reminder here to update all workflows that depend on this workflow?

Reply:
I think a reminder about dependent workflows is a good idea. But, my preference would to not add documentation or instructions about uploading a token as a secret. There is plenty of help on the web for that with a quick google search. I am worried about broken future links or outdated instructions.

Reply:
I think the code cov link is probably sufficient for how to add a secret: https://docs.codecov.com/docs/adding-the-codecov-token, but I think this can be added to ghactions4r instead of FIMS?

I don't think there are any dependent workflows on this one.